.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Mainnet or testnet?**
+Which network you are using the app with.
```

Comment:
I would just add a link to the organization

Reply:
@sohkai done